mitodl/micromasters
# Add support for discontinued runs (#5064)

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #5064 

#### What's this PR do?
This PR reconfigures the data models and queries to ignore discontinued course runs so the application behaves as if they don't exist, except for django admin that can still see them

#### How should this be manually tested?
- Mark a current run as discontinued in django admin
- The run should no longer show up in the dashboard
- Create a new run that is future, the dashboard should reflect the future run
- Change that run to be current, the dashboard should let the learner enroll now
